src/data/decks.json

```
@@ -25,6 +25,7 @@
     "tags": [
       "REGULAR"
     ],
+    "nerfed": "07-2021",
```

```suggestion
    "nerfed": "07/2021",
```
Always slashes for dates, not hyphens. :)